src/Temporalio/Common/RetryPolicy.cs

```diff
@@ -1,6 +1,7 @@
 using System;
 using System.Collections.Generic;
 using Google.Protobuf.WellKnownTypes;
+using Temporalio.Exceptions;
```

Comment:
Can you qualify the name in the docs instead of add this `using` that is only referenced in docs?

Reply:
Sure.